etc/transfer-script.sh

```diff
@@ -2,10 +2,12 @@
 # transfer script example
 # /etc/archivematica/automation-tools/transfer-script.sh
 cd /usr/lib/archivematica/automation-tools/
-/usr/share/python/automation-tools/venv/bin/python -m transfers.transfer
-  --user <user> 
-  --api-key <apikey>  
-  --ss-user <user> 
-  --ss-api-key <apikey> 
-  --transfer-source <transfer_source_uuid> 
+/usr/share/python/automation-tools/venv/bin/python -m transfers.transfer \
+  --am-url <archivematica_url> \
+  --ss-url <storage_service_url> \
+  --user <user> \
+  --api-key <apikey> \ 
+  --ss-user <user> \
+  --ss-api-key <apikey> \
```

Comment:
Would it make sense to prefix these example placeholders such as `<archivematica_user>`, `<archivematica_apikey>`, `<storage_service_user>` and `<storage_service_apikey>` like the `url` ones?

Reply:
Good call! I'll do that.